deploy/storageautoscaler.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: ocs.openshift.io/v1
+kind: StorageAutoScaler
+metadata:
+  name: ssdscaler
+  namespace: openshift-storage
+spec:
+  storageCluster:
+    name: ocs-storagecluster
+  deviceClass: ssd
+  storageCapacityLimit: 100Ti
+  maxOsdSize: 220Gi
+  storageScalingThresholdPercent: 1
+  timeoutSeconds: 100
+
+# ---
```

Comment:
Remove these comments?

Reply:
Would remove currently needed for testing

Reply:
If it's needed for testing, why is it commented out?

Reply:
Yes we can uncomment it, but the race will not be needed as an example once this pr got merged

Reply:
If there are temporary changes, how about adding all caps comment when there is something you expect to remove before merge? Then it is more clear during code review so you don't keep getting comments on removing it.